KarhooUISDK/Extensions/String+Extensions.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  String+Extensions.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 26/01/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    var isNotEmpty: Bool { return !self.isEmpty}
```

Comment:
Worried we won't remember to use it since we already have `isEmpty` out of the box